src/features/identityCheck/errors/eduConnect/__test__/EduConnectErrorBoundary.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { EduConnectErrorBoundary } from 'features/identityCheck/errors/eduConnect/EduConnectErrorBoundary'
+import { EduConnectError } from 'features/identityCheck/errors/eduConnect/types'
+import { eventMonitoring } from 'libs/monitoring'
+import { render } from 'tests/utils'
+
+jest.mock('react-query')
+
+describe('EduConnectErrorBoundary component', () => {
+  it('should not monitor error on sentry is error is managed', () => {
```

Comment:
je ne comprends pas la règle

Reply:
je change en "should not monitor error on sentry is error is an expected EduConnectError"